app/vmagent/remotewrite/streamaggr.go

```diff
@@ -0,0 +1,155 @@
+package remotewrite
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/fasttime"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/flagutil"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/prompbmarshal"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/streamaggr"
+	"github.com/VictoriaMetrics/metrics"
+)
+
+var (
```

Comment:
Could you please document all the new flag in vmagent.md? And provide some description over the global stream aggregation concept and why it is needed? Similarly to how we describe `remoteWrite.relabelConfig`

Reply:
I think we also should warn the users about having the same aggregation intervals if a chained aggregation config (global => per_url) is used.